python/pyspark/sql/types.py

```diff
@@ -1433,7 +1433,7 @@ class Row(tuple):
     ``key in row`` will search through row keys.
 
     Row can be used to create a row object by using named arguments.
-    It is not allowed to omit a named argument to represent the value is
+    It is not allowed to omit a named argument to represent that the value is
```

Comment:
I think it was correct to omit `that` as well.

Reply:
yes, we can omit `that`, but adding it makes the sentence a little bit more explanatory.

Reply:
@sharifahmad2061 I just merged this time but let's don't make unnecessary changes like this next time.